src/extension/features/accounts/auto-mark-as-cleared/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'AutomaticallyMarkAsCleared',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Automatically mark transaction as cleared',
```

Comment:
if you could capitalize each word except "as" that would match how the rest of the names look on our settings page :D